tests/Test.ADAL.NET.Unit.net45/UnitTests.cs

```diff
@@ -117,6 +119,30 @@ public void AuthorityTypeDetectionTest()
             Assert.AreEqual(AuthorityType.ADFS, Authenticator.DetectAuthorityType("https://abc.com/adfs/dummy/"));
         }
 
+        [TestMethod]
```

Comment:
I like to add a comment like "Test for bug XXX and the github issue url". Bugs have a lot of context..